cli/printer/printer_test.go

```diff
@@ -381,140 +341,154 @@ func TestPrinter_Assessment(t *testing.T) {
 		// There is also something else a bit weird, namely it uses `groups`
 		// which is not a child field of the `user` resource. I'd love to restore
 		// these.
-		// 		{
-		// 			"users.all(groups.none(gid==0))\n",
-		// 			`[failed] users.all()
-		//   actual:   [
-		//     0: user {
-		//       uid: 0
-		//       groups.list: [
-		//         0: user group id = group/0/root
-		//         1: user group id = group/1/bin
-		//         2: user group id = group/1000/chris
-		//       ]
-		//       gid: 0
-		//       groups: groups
-		//       name: "root"
-		//     }
-		//     1: user {
-		//       uid: 1
-		//       groups.list: [
-		//         0: user group id = group/0/root
-		//         1: user group id = group/1/bin
-		//         2: user group id = group/1000/chris
-		//       ]
-		//       gid: 1
-		//       groups: groups
-		//       name: "bin"
-		//     }
-		//     2: user {
-		//       uid: 1000
-		//       groups.list: [
-		//         0: user group id = group/0/root
-		//         1: user group id = group/1/bin
-		//         2: user group id = group/1000/chris
-		//       ]
-		//       gid: 1000
-		//       groups: groups
-		//       name: "chris"
-		//     }
-		//     3: user {
-		//       uid: 1001
-		//       groups.list: [
-		//         0: user group id = group/0/root
-		//         1: user group id = group/1/bin
-		//         2: user group id = group/1000/chris
-		//       ]
-		//       gid: 1000
-		//       groups: groups
-		//       name: "christopher"
-		//     }
-		//   ]
-		// `,
-		// 		},
+		// {
+		// 	"users.all(groups.none(gid==0))\n",
+		// 	strings.Join([]string{
+		// 		"[failed] users.all()",
+		// 		"  actual:   [",
+		// 		"    0: user uid=0 gid=0 name=\"root\" {",
+		// 		"      groups.list: [",
+		// 		"        0: user group id = group/0/root",
+		// 		"        1: user group id = group/1001/chris",
```

Comment:
Am I right to assume that this child list still prints them in the old way? (feel free to resolve this comment if so, I can see that this is a follow-up if that's the case)

Reply:
No, the child list is completely missing:
```
cnquery run -c "users.all(groups.none(gid==0))" 
→ no provider specified, defaulting to local. Use --help to see all providers.
→ loaded configuration from /etc/opt/mondoo/mondoo.yml using source default
[failed] users.all()
  actual:   [
    0: user uid=0 gid=0 name="root" 
    1: user uid=1 gid=1 name="daemon"
```

But it looks like this query isn't evaluated correctly, at least not as I read it. When you take a look at the `daemon` user:
```
id daemon 
uid=1(daemon) gid=1(daemon) groups=1(daemon)
```

And cnquery v9.0.2 also does not show the list:
```
cnquery run -c "users.all(groups.none(gid==0))" 
→ no provider specified, defaulting to local. Use --help to see all providers.
→ loaded configuration from /etc/opt/mondoo/mondoo.yml using source default
[failed] users.all()
  actual:   [
    0: user {
      uid: 0
      gid: 0
      name: "root"
    }
    1: user {
      uid: 1
      gid: 1
      name: "daemon"
    }
...
```

There is something strange here, but it seems to be independent of this PR.

Ok, with this PR, I changed the expected output. But the tests were commented out with https://github.com/mondoohq/cnquery/pull/1374. So, how we return these child lists has changed during the v9 migration. I'd opt for solving this in a follow-up.

Reply:
I think I mentioned this before: The `users` resource does not have a `groups` field. This is what is likely causing the behavior